lib/lintSource.js

```diff
@@ -64,9 +71,11 @@ module.exports = function lintSource(stylelint, options = {}) {
 		});
 
 		return getConfig.then((result) => {
-			// TODO TYPES possible null
-			const config =
-				/** @type {{ config: import('stylelint').StylelintConfig, filepath: string }} */ (result).config;
+			if (!result) {
+				throw new Error('Not found a config file!');
```

Comment:
Maybe `Config file not found!` is better?

Reply:
I think we can lose the exclamation point too.